packages/url/src/filter-url-for-display.js

```diff
@@ -21,8 +21,8 @@ export function filterURLForDisplay( url, maxLength = null ) {
 		filteredURL = filteredURL.replace( '/', '' );
 	}
 
-	const mediaRegexp = /([\w|:])*\.(?:jpg|jpeg|gif|png|svg)/;
-
+	const mediaRegexp =
+		/([\w|:])*\.(?:jpg|jpeg|gif|png|svg|ico|webp|mp3|m4a|ogg|wav|mp4|m4v|mov|wmv|avi|mpg|ogv|3gp|3g2)/;
```

Comment:
I didn't include [the documents](https://codex.wordpress.org/Uploading_Files) in the accepted file types, as this isn't a problem with the File block. But we could always add those to future-proof a bit.

Reply:
Feel free to add them all IMHO.